src/index.html

```diff
@@ -13,6 +13,144 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section>
+      <div class="stars stars--0">
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+        <span
+          class="stars__star"
+          aria-label="Inactive star"
+        ></span>
+      </div>
```

Comment:
There is no empty line between sibling blocks of HTML. As per the checklist, you should add empty lines between multiline sibling blocks of HTML to improve readability. This should be applied to all sibling divs in this file.